jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpGenerator.java

```diff
@@ -68,25 +73,14 @@ public enum Result
         DONE                    // The current phase of generation is complete
     }
 
-    // other statics
-    public static final int CHUNK_SIZE = 12;
-
     private State _state = State.START;
     private EndOfContent _endOfContent = EndOfContent.UNKNOWN_CONTENT;
     private MetaData _info;
-
     private long _contentPrepared = 0;
     private boolean _noContentResponse = false;
     private Boolean _persistent = null;
-
-    private static final Index<Boolean> ASSUMED_CONTENT_METHODS = new Index.Builder<Boolean>()
-        .caseSensitive(false)
-        .with(HttpMethod.POST.asString(), Boolean.TRUE)
-        .with(HttpMethod.PUT.asString(), Boolean.TRUE)
-        .build();
-
-    // data
     private boolean _needCRLF = false;
+    private int _maxHeaderBytes;
```

Comment:
I wonder if 0 is a sensible default, even if we always call the setter before using the generator.

Reply:
We did not check for response headers max size before, so `0` keeps the original behavior.